frontend/src/metabase-lib/v1/parameters/utils/operators.ts

```diff
@@ -64,7 +64,7 @@ function getParameterOperatorType(parameterType?: string) {
 
 export function buildTypedOperatorOptions(
   operatorType: OperatorType,
-  sectionId: string,
+  sectionId: "number" | "string" | "date" | "location" | "id",
```

Comment:
```suggestion
  sectionId: SectionId,
```

Reply:
Is there any other place where we can define `SectionId` to not hit this problem?

Reply:
inside the metabase-lib 😄 let me fix it

Reply:
[`6b5f496` (#42269)](https://github.com/metabase/metabase/pull/42269/commits/6b5f49624f4a6cb5d959a705f4428a0547b7769a)